github3/repos/repo.py

```diff
@@ -1282,6 +1282,32 @@ def imported_issue(self, imported_issue_id):
         return self._instance_or_null(ImportedIssue, json)
 
     @requires_auth
+    def imported_issues(self, number=-1, since=None, etag=None):
+        """Retrieve imported issues
+
+        :param int number: (optional), number of imported issues to return.
+            Default: -1 returns all branches
+        :param since: (optional), Only commits after this date will
+            be returned. This can be a `datetime` or an `ISO8601` formatted
```

Comment:
Should have double back-ticks around datetime and ISO8601 for reStructuredText to format them the way you want it to. :)